presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -491,14 +491,14 @@ JMX queries to get the metrics and verify the cache usage::
 
     SELECT * FROM jmx.current."com.facebook.presto.hive:name=iceberg_parquetmetadata,type=cachestatsmbean";
 
-Metastore Versioned Cache
+Metastore Cache
 ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^
```
Formatting line should be the same numbers of characters in the line above it.

Reply:
Thanks, updated.